binderhub/templates/index.html

```diff
@@ -14,6 +14,7 @@
 {% block main %}
 <div id="main" class="container">
   <div class="row">
+    <h3 style="margin-top: -60px;text-align: center;">(beta)</h3>
```

Comment:
Can we avoid inline styles? The bootstrap css class "text-center" applies the text-align bit. CSS can be used for the margin adjustment.

Reply:
Sounds good - I'll change this yuvi when runs me through deploying changes via PRs today

Reply:
+1 on what @minrk said